flood_api/dependencies/flooddata.py

```diff
@@ -0,0 +1,62 @@
+import asyncio
+from datetime import timezone
+from typing import Annotated
+
+import geopandas as gpd
+import pandas as pd
+from aiocron import crontab
+from fastapi import FastAPI, Request, Depends
+from shapely import wkt
+
+from flood_api.settings import settings
+
+
+def fetch_parquet(path) -> gpd.GeoDataFrame:
+    print(f"Reloading data from {path}")
+
+    df = pd.read_parquet(
+        path,
+        engine="fastparquet",
+        storage_options={"anon": True},
+    )
+
+    # Convert WKT strings to geometry objects
+    df["geometry"] = df["wkt"].apply(wkt.loads)
+
+    gdf = gpd.GeoDataFrame(df, geometry="geometry").drop(columns="wkt")
+
+    print(f"Done reloading data from {path}")
```

Comment:
If this is meant to be logging, we should change this print to use the logging library.

Reply:
Agreed